ebu_tt_live/carriage/__init__.py

```diff
@@ -0,0 +1,2 @@
+from filesystem import FileSystemCarriageImpl
```

Comment:
Do we now have a single init file that imports everything for all carriage mechanisms even if they are not actually used? Seems undesirable.